### PR TITLE
removed lorem ipsum

### PR DESCRIPTION
**Pull request by homelessbirds**
I will write a better description

### CODE REVIEW

File: src/content/docs/practice-folder/lorem.mdx [hunk 1]
```diff
@@ -9,14 +9,10 @@ Alice was beginning to get very tired of sitting by her sister on the bank, and
...
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam lobortis luctus lobortis. Mauris eget volutpat dui. In ultricies molestie ante at tempor. Cras bibendum facilisis venenatis. Aenean augue urna, volutpat ac commodo eu, viverra vitae augue. Nam ullamcorper vehicula lacus, at dignissim velit sollicitudin a. Nullam sed arcu sem. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Nullam interdum mauris interdum lectus faucibus, id ullamcorper purus gravida. Aliquam sollicitudin maximus justo, a semper quam mollis vel. Nulla mattis quam eget varius venenatis. Fusce arcu tellus, faucibus efficitur malesuada in, sollicitudin quis velit. Nunc vitae urna porta, dapibus lectus a, pellentesque tellus. Nunc dolor orci, euismod non vestibulum et, accumsan ac libero. Donec nec porta arcu, ac placerat nisi. Aliquam ornare eros augue, vel convallis odio feugiat sit amet.
```
Comment (paperclypse) [inline src/content/docs/practice-folder/lorem.mdx:12]: Are we sure this is lorem ipsum?

File: src/content/docs/practice-folder/lorem.mdx [hunk 2]
```diff
@@ -9,14 +9,10 @@ Alice was beginning to get very tired of sitting by her sister on the bank, and
...
 
 In another moment down went Alice after it, never once considering how in the world she was to get out again.
 
-Donec nec tempor arcu. Maecenas facilisis ultrices finibus. Curabitur tempor quam sit amet lectus venenatis, vitae scelerisque lectus facilisis. Suspendisse facilisis nisi eu diam maximus ullamcorper. Suspendisse a mauris pharetra, laoreet leo in, volutpat leo. Phasellus cursus cursus lectus. Vestibulum aliquet, sapien sit amet dictum dapibus, ipsum nibh tincidunt leo, eget congue nulla turpis nec tortor. Nulla sed sem rhoncus, aliquam lacus ut, placerat elit. Etiam a dolor ultricies, condimentum metus quis, lobortis ipsum. Sed quis metus non tortor tempor pellentesque. Aenean in elit posuere, hendrerit risus a, mollis ipsum. Duis vel elit erat. Praesent est quam, vulputate a lacus in, porta feugiat diam. Praesent euismod eros ac sem molestie gravida. Vivamus vel dictum mi, ut imperdiet quam. Aenean molestie nunc vitae urna cursus dignissim eu nec diam.
```
Comment (paperclypse) [inline src/content/docs/practice-folder/lorem.mdx:18]: How did all this junk get in here?

Comment (paperclypse) [reply src/content/docs/practice-folder/lorem.mdx:18]: Thanks for cleaning it up!

Comment (homelessbirds) [reply src/content/docs/practice-folder/lorem.mdx:18]: Thanks for the feedback! Very constructive.

File: src/content/docs/practice-folder/lorem.mdx [hunk 3]
```diff
@@ -25,14 +21,10 @@ Either the well was very deep, or she fell very slowly, for she had plenty of ti
 
 Down, down, down. Would the fall never come to an end? “I wonder how many miles I’ve fallen by this time?” she said aloud. “I must be getting somewhere near the centre of the earth. Let me see: that would be four thousand miles down, I think—” (for, you see, Alice had learnt several things of this sort in her lessons in the schoolroom, and though this was not a very good opportunity for showing off her knowledge, as there was no one to listen to her, still it was good practice to say it over) “—yes, that’s about the right distance—but then I wonder what Latitude or Longitude I’ve got to?” (Alice had no idea what Latitude was, or Longitude either, but thought they were nice grand words to say.)
```
Comment (paperclypse) [inline src/content/docs/practice-folder/lorem.mdx]:
```suggestion
Down, down, down. Would the fall never come to an end? “I wonder how many miles I’ve fallen by this time?” she said aloud. “I must be getting somewhere near the centre of the earth. Let me see: that would be four thousand miles down, I think—”  “—yes, that’s about the right distance—but then I wonder what Latitude or Longitude I’ve got to?” 
```

Parentheticals are optional, right?

**Pull request closed without merging**